examples/forwarding-refs/log-props-after.js

```diff
@@ -0,0 +1,26 @@
+function logProps(WrappedComponent) {
+  class LogProps extends React.Component {
+    componentWillReceiveProps(nextProps) {
```

Comment:
I think we'd better use `componentDidUpdate` instead as `componentWillReceiveProps` is deprecated then

Reply:
Nice catch! Updated.